mbs_results/calculate_ratio_estimation.py

```diff
@@ -1,24 +1,20 @@
-import pandas as pd
+import numpy as np
 
 
 def calculate_ratio_estimation(
```

Comment:
Can we please update docstrings ? 11 args in docstrings but 9 actual args, ( i think the order is fine)